phpunit.xml.dist

```diff
@@ -0,0 +1,7 @@
+<phpunit colors="true">
```

Comment:
With bare skeletal functionality, that would be redundant since phpunit/phpunit/phpunit shipped in the vendor directory to execute the tests does this. It populates as far as I understand, it populates PHPUNIT_COMPOSER_INSTALL which is then used by phpunit framework to autoload classes.

Since in ZF3 we do not use zend autoloader, bootstrapping framework there is also not needed.

However, if want to customize test runner more, maybe with some utility functions or reusable mocks to help bring up application as a common place for setUp() then it can be created as an empty stub ahead of time open to those extensions. Would that be beyond the scope of this or we stick to what works bare minimum?